docs/CHANGELOG.rst

```diff
@@ -24,13 +24,18 @@ Changed
   - ``cufflinks``
   - ``cuffnorm``
   - ``cuffquant``
+- Update ``workflow-bbduk-star-htseq``
```

Comment:
Change log needs to be updated promptly with each commit. Don't put all entries in the last commit.

Reply:
I will leave it as it is for this time, since I am very short with time, but will keep in mind for next time.